src/cdk/tree/tree-errors.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * Returns an error to be thrown when there are multiple nodes that are missing a when function.
+ * @docs-private
+ */
+export function getTreeMultipleDefaultNodeDefsError() {
+  return Error(`cdk-tree: There can only be one default row without a when predicate function.`);
```

Comment:
Heads up that I recently removed the prefix `cdk-table` from the errors since I realized it was clear that it was being thrown from the table and it didn't make sense for users of `mat-table` to see `cdk-table`